upkie/envs/wrappers/tests/random_push_test.py

```diff
@@ -0,0 +1,53 @@
+#!/usr/bin/env python3
+# -*- coding: utf-8 -*-
+#
+# SPDX-License-Identifier: Apache-2.0
+# Copyright 2024 Inria
+
+"""Test RandomPush wrapper."""
+
+import unittest
+
+import gymnasium
+import numpy as np
+
+from upkie.envs.wrappers.random_push import (
+    RandomPush,
+)
+from upkie.envs.wrappers.tests.envs import ConstantObservationEnv
+
+
+class RandomPushTestCase(unittest.TestCase):
+    def test_wrapper(self):
+        env = ConstantObservationEnv(1.0)
+        env.get_spine_action = lambda action: {}
+        wrapped_env = RandomPush(
+            env,
+            push_prob=1,
+            push_generator=lambda: np.array([42, 42, 42])
+            )
+        action = np.array([1.0])
+        spine_action = wrapped_env.get_spine_action(action)
+        self.assertTrue("bullet" in spine_action)
+        assert "external_forces" in spine_action["bullet"]
```

Comment:
If you agree with using `self.assert*` we should replace the other asserts as well :wink: